SETUP.md

```diff
@@ -15,7 +15,16 @@ Before you start using this template, complete the following steps:
   - [ ] Add the project name
   - [ ] Add the missing information based on the comments in the file
 - [ ] Add code owners to the `.github/CODEOWNERS` file if applicable
+- [ ] Define a changelog strategy[^1]
 - [ ] Delete the `SETUP.md` file
+
+### Project information
+
+If your project needs to provide additional information like backward compatibility, system status or similar, include it to the `README.md` file of additional file in a clear and informative format (e.g. compatibility matrix, table, timeline, etc.). Also, if there is a need for additional information for contributors, additional security measures or specific license, update and adjust related files.
```

Comment:
What is "system status" referring to?

Reply:
Is the system running?
I guess this makes more senes for backend projects

Reply:
I would also remove this reference then, it's just confusing within the context of Android